static-site/app/blog/parseMarkdown.ts

```diff
@@ -6,11 +6,15 @@ import { siteUrl } from "../../data/BaseConfig";
 export default async function parseMarkdown({
   mdString,
   addHeadingAnchors = false,
+  headingAnchorClass = "",
 }: {
   mdString: string
 
   /** Should `<a>` tags be added to headings? */
   addHeadingAnchors?: boolean
+
+  /** Class name for heading anchors (from page-specific CSS Module) */
+  headingAnchorClass?: string
```

Comment:
does it ever make sense to call `parseMarkdown({ mdstring, addHeadingAnchors: true })`, i.e., without providing a `headingAnchorClass`? Since that's only going to be used when `addHeadingAnchors` is `true`, I wonder if there's a better default than `""`, so that if somebody forgets it, we don't end up with `class=""` in the output?

This is probably nitpicky.

Reply:
Valid case. How about error?

https://github.com/nextstrain/nextstrain.org/blob/cf053361645509cd488ec421fc27ea7840313881/static-site/app/blog/parseMarkdown.ts#L20-L22

It shows a client-side error

![image](https://github.com/user-attachments/assets/780fb751-09a9-4881-8ce3-dd31926ca100)

and server-side error

```
[verbose]       next.js handling request for "/blog/2024-12-10-auspice-1password-issue"
Error parsing markdown: Error: parseMarkdown: headingAnchorClass must be defined when addHeadingAnchors is true.
```